mlxbf-bootctl.c

```diff
@@ -73,8 +73,40 @@ typedef struct  __attribute__((__packed__)) {
   uint8_t unused[3];
 } psc_app_img_hdr_t;
 
+/* Post reset watchdog SMC parameters */
+#define POST_RESET_WDOG_INTERVAL_SHIFT 0
+#define POST_RESET_WDOG_INTERVAL_MASK 0xFFFF
+#define POST_RESET_WDOG_MODE_SHIFT 16
+#define POST_RESET_WDOG_MODE_MASK 0xFF
+
+/*
+ * Boot watchdog (ARM SBSA) modes of operation.
+ *
+ * Disabled: watchdog will not be armed for boot (unless used for boot swap).
+ * Standard: watchdog will be armed at boot start and refreshed periodically.
+ * Time Limit: watchdog will be armed at boot start and never refreshed. In
+ *             special cases (like booting in livefish mode or entering UEFI
+ *             menu) the watchdog will be disabled.
+ */
+#define BF_BOOT_WDOG_MODE_DISABLED 0
+#define BF_BOOT_WDOG_MODE_STANDARD 1
+#define BF_BOOT_WDOG_MODE_TIME_LIMIT 2
+#define BF_BOOT_WDOG_MODE_MAX BF_BOOT_WDOG_MODE_TIME_LIMIT
+
+/* Boot watchdog mode strings */
+#define BF_BOOT_WDOG_MODE_DISABLED_STR "disabled"
+#define BF_BOOT_WDOG_MODE_STANDARD_STR "standard"
+#define BF_BOOT_WDOG_MODE_TIME_LIMIT_STR "time_limit"
+#define BF_BOOT_WDOG_MODE_UNKNOWN_STR "unknown"
```

Comment:
Maybe we could consider an array to match the enum struct?
const char * const bf_boot_wdog_mode_str[] = {"disable", "standard", "time_limit", "unknown"}